src/locales/fr/translation.json

```diff
@@ -1,5 +1,11 @@
 {
   "welcome_message": "Bienvenue dans l'API de l'application Ecommerce de Cogito",
+  "disactvated": "votre statut a été désactivé",
+  "fail": "utilisateur non trouvé",
+  "failId": "aucun utilisateur trouvé avec cet ID",
+  "retrievedAll": "tous les utilisateurs récupérés",
+  "retrieved": "utilisateur récupéré",
+  "updateStatus": "statut mis à jour avec succès",
```

Comment:
- use snake case instead and remove fail because fail means a different thing. I would suggest casting the key to what the message says
- fail is not meaning full to the message you are responding.
- here is an example of what you trying to do which is correct "user_not_found": "utilisateur non trouvé".